src/corelibs/U2View/src/ov_sequence/GSequenceGraphView.cpp

```diff
@@ -94,82 +87,74 @@ GSequenceGraphView::GSequenceGraphView(QWidget *p, SequenceObjectContext *ctx, G
 }
 
 void GSequenceGraphView::setGraphDrawer(GSequenceGraphDrawer *gd) {
+    SAFE_POINT(graphDrawer == nullptr, "GSequenceGraphDrawer was already set", );
     graphDrawer = gd;
+    connect(graphDrawer, SIGNAL(si_graphDataUpdated()), renderArea, SLOT(update()));
     update();
 }
 void GSequenceGraphView::mousePressEvent(QMouseEvent *me) {
     setFocus();
 
     if (me->modifiers() == Qt::ShiftModifier && me->button() == Qt::LeftButton) {
-        float pos = toRenderAreaPoint(me->pos()).x() / renderArea->getCurrentScale() + getVisibleRange().startPos;
-        addLabel(pos);
+        float sequencePos = toRenderAreaPoint(me->pos()).x() / renderArea->getCurrentScale() + getVisibleRange().startPos;
+        int capturingDistancePx = 4;
```

Comment:
What's the magic number? At least make in `static constexpr`

Reply:
it is not a magic number - it has a dedicated name